monitor/worker.js

```diff
@@ -19,13 +21,12 @@ parentPort.on('message', async _ => {
     await scenario.run({ page, baseURL: process.env.TARGET_URL });
 
     const end = performance.now();
-    const duration = end - start;
+    const duration = Math.floor(end - start);
 
     const metric = scenario.name.toLowerCase().replace(/ /g, '.');
 
-    client.gauge(metric, Math.floor(duration));
-
-    parentPort.postMessage(`Scenario: "${scenario.name}" took ${Math.floor(duration)}ms`);
+    client.gauge(metric, duration);
+    logger.info({metric, duration});
```

Comment:
Are these logs going anywhere?  Do we record the STD out of this process/image somewhere?

Reply:
See the `log_configuration` is set to json where it is normally set to raw: https://github.com/mbta/devops/pull/1544/files